needle/modules/configuration/mdm_pull.py

```diff
@@ -0,0 +1,71 @@
+from core.framework.module import BaseModule
+from core.device.device import Device
+from core.utils.menu import choose_from_list_data_protection, choose_from_list, choose_boolean
+from core.utils.utils import Utils
+import re, sys
+
+class Module(BaseModule):
+    meta = {
+        'name': 'MDM Pull',
+        'author': 'Oliver Simonnet (@MWRLabs)',
+        'description':  'Pulls the Effective Configuration from device.',
```

Comment:
Effective configuration is the configuration abided by, by the device.
I will rephrase for clarity.